app/models/miq_report/generator/trend.rb

```diff
@@ -67,7 +67,7 @@ def build_calculate_trend_point(rec, col)
     begin
       val = MiqStats.solve_for_y(rec.send(CHART_X_AXIS_COLUMN_ADJUSTED).to_i, @trend_data[col][:slope], @trend_data[col][:yint])
       return val > 0 ? val : 0
-    rescue ZeroDivisionError => err
+    rescue ZeroDivisionError => _err
```

Comment:
If you aren't using it, there's no reason to even underscore it. (Just `rescue ZeroDivisionError`)